packages/tracing/src/hubextensions.ts

```diff
@@ -19,25 +20,68 @@ function traceHeaders(this: Hub): { [key: string]: string } {
 }
 
 /**
- * Use RNG to generate sampling decision, which all child spans inherit.
+ * Use sample rate (given in options as either a constant or ) along with a random number generator to make a sampling
```

Comment:
> constant or

? 😅

Reply:
🤔 

Good question.

Reply:
Oh, heh. Incomplete editing. (Moved that parenthetical to its own sentence on the next line, but missed getting rid of it once I had.)

Thanks for catching. Fixed.